tests/functional/a/access/access_to_protected_members.py

```diff
@@ -85,9 +85,9 @@ def __le__(self, other):
         Test a correct access as the access to protected member
         is inside a special method even if it is deeply nested
         """
-        if 2 > 1:
+        if 2 > 1:  # pylint: disable=comparison-of-two-literals
```

Comment:
I'm not suggesting the name change everywhere, an IDE search and replace will be more efficient.